css-layout-classes/styles.css

```diff
@@ -0,0 +1,41 @@
+#container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  align-items: center;
+  justify-content: center;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex: none;
+}
+
+.column-one-third {
+  flex-basis: calc(100%/3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100%/4);
+}
+
+.column-two-third {
+  flex-basis: 2/3;
+}
+
+.width {
```

Comment:
you want to use width instead of max-width since max width is a limiting property, its not actually assigning a width. Also, width needs to be on all of the images.